packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -298,6 +300,25 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
             }
         });
 
+        commands.registerCommand({ id: 'see-whether-order-matters', label: 'Try opening the same file with different options!' }, {
+            execute: async () => {
+                const workspaceRoot = this.workspaceService.tryGetRoots()[0];
+                const firstEditor = await this.widgetManager.getOrCreateWidget('code-editor-opener', {
+                    'uri': workspaceRoot.resource.resolve('package.json').toString(),
+                    'kind': 'navigatable',
+                    'counter': 0
+                });
+                const secondEditor = await this.widgetManager.getOrCreateWidget('code-editor-opener', {
+                    'kind': 'navigatable',
+                    'uri': workspaceRoot.resource.resolve('package.json').toString(),
+                    'counter': 0,
+                });
+                console.log('SENTINEL FOR WHETHER THE EDITORS ARE EQUAL...', firstEditor === secondEditor);
+                firstEditor.dispose();
+                secondEditor.dispose();
+            }
+        });
```

Comment:
Should we have a place where we can put those commands without polluting random extensions? Maybe `@theia/api-samples` would be a better place for that next time?

Reply:
If it's going to be dropped, I just put it in whatever package seems to have already imported most of what I need 😄.